app/ui/browser-modern/views/browser/page/index.jsx

```diff
@@ -111,10 +111,8 @@ class Page extends Component {
     });
 
     this.webview.addEventListener('page-title-set', e => {
+      // Not emitted when pages don't have a title.
       this.props.dispatch(PageActions.setPageDetails(this.props.pageId, { title: e.title }));
-      // Make sure we record this page navigation in the remote history
-      // only after we have a title available.
-      this.props.dispatch(ProfileEffects.addRemoteHistory(this.props.pageId));
     });
 
     this.webview.addEventListener('page-favicon-updated', e => {
```

Comment:
This isn't in the favicon event listener, it's in the dom-ready listener. Git diff failing :)